discovery-infra/start_discovery.py

```diff
@@ -149,6 +158,45 @@ def _secondary_tfvars(master_count, nodes_details, machine_net):
         }
 
 
+def _generate_static_ips_data(num_masters, num_workers, machine_net):
```

Comment:
can we move it to , static_ip_tools.py ?

Reply:
fixed